### PR TITLE
14850 - Allows transactions to be seen in the left bar in account-info

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/14850
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/views/auth/AccountSettings.vue [hunk 1]
```diff
@@ -172,7 +172,7 @@
                 role="listitem"
                 :to="transactionUrl"
                 data-test="transactions-nav-item"
-                v-if="isPremiumAccount || isSbcStaffAccount || isStaffAccount"
+                v-if="isPremiumAccount || isSbcStaffAccount || isStaffAccount || isBasicAccount"
```
Comment (kialj876) [inline auth-web/src/views/auth/AccountSettings.vue:175]: Hey Jia, these are all the account types we have in prod so you can remove this v-if

Comment (Jxio) [reply auth-web/src/views/auth/AccountSettings.vue:175]: okay, thanks!

File: auth-web/tests/unit/components/Product.spec.ts [hunk 2]
```diff
@@ -159,19 +159,18 @@ describe('Product.vue', () => {
     expect(wrapper.vm.hasDecisionNotBeenMade).toBeFalsy()
   })
 
-  it('premium product should be disabled in basic account settings', async () => {
+  it('premium product should be visiable to basic account settings', async () => {
```
Comment (kialj876) [inline auth-web/tests/unit/components/Product.spec.ts:162]: Your changes should not be effecting the Product component. Premium only things in here should still be premium only and have the minus checkboxes visible.

File: auth-web/tests/unit/components/Product.spec.ts [hunk 3]
```diff
@@ -159,19 +159,18 @@ describe('Product.vue', () => {
...
     pprProduct.subscriptionStatus = ProductStatus.NOT_SUBSCRIBED
-    pprProduct.premiumOnly = true
```
Comment (kialj876) [inline auth-web/tests/unit/components/Product.spec.ts:164]: premiumOnly needs to be true on the product for this test to be valid

File: auth-web/tests/unit/components/Product.spec.ts [hunk 4]
```diff
@@ -159,19 +159,18 @@ describe('Product.vue', () => {
...
-    wrapper = wrapperFactory({ productDetails: pprProduct, isexpandedView: false, isAccountSettingsView: true, isBasicAccount: true })
+    wrapper = wrapperFactory({ productDetails: pprProduct, isexpandedView: false, isAccountSettingsView: true, isBasicAccount: false })
```
Comment (kialj876) [inline auth-web/tests/unit/components/Product.spec.ts:164]: isBasicAccount needs to be true for this test to be valid

**Review by kialj876 — CHANGES_REQUESTED** [review]
Hey Jia, I think the test change you made needs to be reverted (it's testing a valid use case)

File: auth-web/src/components/auth/mixins/AccountMixin.vue [hunk 5]
```diff
@@ -52,5 +52,9 @@ export default class AccountMixin extends Vue {
   protected get isSbcStaffAccount (): boolean {
     return this.currentOrganization?.orgType === Account.SBC_STAFF
   }
+
+  protected get isBasicAccount (): boolean {
```
Comment (seeker25) [inline auth-web/src/components/auth/mixins/AccountMixin.vue:56]: Is this used?

Comment (Jxio) [reply auth-web/src/components/auth/mixins/AccountMixin.vue:56]: it used before I removed account type check, I'll move it.

**Comment by seeker25** [comment]
Please update package.json version and package-lock.json version.

**Comment by seeker25** [comment]
Can you update the package.json to refer to the new sbc-common-components as well?

**Comment by Jxio** [comment]
> Can you update the package.json to refer to the new sbc-common-components as well?
> 
> ` "3.0.7",`

yes, it's updated now

**Review by seeker25 — APPROVED** [review]

**Comment by seeker25** [comment]
@Jxio we can merge this when https://github.com/bcgov/sbc-pay/pull/1103 is done.

**Review by kialj876 — APPROVED** [review]

**Pull request merged**